cmd/boots/main.go

```diff
@@ -68,8 +68,12 @@ type config struct {
 	ipxeHTTPEnabled bool
 	// ipxeRemoteTFTPAddr is the address of the remote TFTP server serving iPXE binaries
 	ipxeRemoteTFTPAddr string
-	// remoteTFTPPort is the address of the remote HTTP server serving iPXE binaries
+	// ixpeRemoteHTTPAddr is the address and port of the remote HTTP server serving iPXE binaries
```

Comment:
was this intentional?

Reply:
Yeah, that was fixing a comment that was incorrect.

Reply:
oh doh, for some reason i did even see the `//`. sorry. Does look like the word is misspelled?

Reply:
Fixed.